### PR TITLE
[LA-442]: Adding pipeline permissions

### PR DESCRIPTION
**Pull request by sergeychernyshov-code**
## What does this change?
changes credentials to oidc instead of aws keys, in this PR we are adding exclusive pipeline permissions needed for OIDC

## Why?
Pipelines with OIDC require defined set of permissions

## Link to supporting ticket or Screenshots (if applicable)
https://axelspringer.atlassian.net/browse/LA-442

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ This repo hosts shared github workflows used by other repositories.
 
 Please refer to githubs documentation on how to use and implement shared workflows: https://docs.github.com/en/actions/using-workflows/reusing-workflows
 
+For authentication to AWS this job uses OIDC instead of plain IAM credential keys.
```
Comment (k-pusch) [inline README.md:7]: Please add a note that this change got introcuded in version 3.

File: README.md [hunk 2]
```diff
@@ -23,16 +35,12 @@ Please refer to githubs documentation on how to use and implement shared workflo
 ### Usage
 
 ```yaml
-
 jobs:
   …
   cleanup-after-merge:
     uses: spring-media/la-shared-github-workflows/.github/workflows/reusable-workflow__aws-remove-bucket.yaml@v1
```
Comment (k-pusch) [inline README.md:41]:
```suggestion
    uses: spring-media/la-shared-github-workflows/.github/workflows/reusable-workflow__aws-remove-bucket.yaml@v3
```

Comment (k-pusch) [reply README.md:41]: Unfortnuately i only can make the suggestion here. But in all the comments we would need to update it to v3 for easier usage in future.

**Review by k-pusch — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sergeychernyshov-code) [reply README.md:7]: added a hint that workflows should use v3

File: README.md [hunk 3]
```diff
@@ -64,8 +70,6 @@ jobs:
       build_script_name: <the npm script to build the project>
```
Comment (k-pusch) [inline README.md:70]: Above again: @v3 instead of v1 please

**Review by k-pusch — CHANGES_REQUESTED** [review]
We still have a bunch of 

spring-media/la-shared-github-workflows/.github/workflows/reusable-workflow__js__deploy-to-s3.yaml@v1

in the documentation, please check again and update everything to v3

File: README.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ This repo hosts shared github workflows used by other repositories.
...
+```yaml
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          audience: sts.amazonaws.com ## Global audience
+          aws-region: eu-central-1 ## Region
+          role-to-assume: arn:aws:iam::481919085934:role/actions-role ## numeric number if Loyalty AWS account where OIDC trust identity is configured
+          role-session-name: github-automated-workflow-session ## Role name in Loyalty AWS account, can be scoped to specific workflow in Trust Relationship if needed
+```yaml
+
```
Comment (k-pusch) [inline README.md:18]: Something is off with the MARKDOWN syntax here. 

The part ###Workflows should not be part of the yaml code.
If you look at this README file something is off.

https://github.com/spring-media/la-shared-github-workflows/blob/17f7a87848b33929468ed5a9a218d8054eae6fe0/README.md

**Review by k-pusch — CHANGES_REQUESTED** [review]

**Review by k-pusch — APPROVED** [review]

**Pull request merged**